app/src/main/res/values/strings.xml

```diff
@@ -86,6 +86,18 @@
         <item><xliff:g example="2 days" id="DAYS">%1$s</xliff:g>, <xliff:g example="2 hours" id="HOURS">%2$s</xliff:g>, and <xliff:g example="2 minutes" id="MINUTES">%3$s</xliff:g></item>
     </string-array>
 
+    <string name="vibration_title">Vibration</string>
+    <string-array name="vibration_entries">
+        <item>Always Off</item>
+        <item>On (by default)</item>
+        <item>Off (by default)</item>
```

Comment:
Could you please add translations for German, Spanish, Italian, Norwegian, Russian and Ukranian? You can use ChatGPT for this, like this:

```
Please translate into German, Spanish, Italian, Norwegian, Russian and Ukranian:
<string name="vibration_title">Vibration</string>
<string-array name="vibration_entries">
        <item>On by default</item>
        <item>Off by default</item>
        <item>Disabled</item>
</string-array>
```

This also is not very intuitive, so probably we should carefully explain to the user what is the deal here.

Reply:
> Could you please add translations for German, Spanish, Italian, Norwegian, Russian and Ukranian? You can use ChatGPT for this, like this:

Sure, I am adding the translations of those languages.

> This also is not very intuitive, so probably we should carefully explain to the user what is the deal here.

Can you please say more on how can I do this?